src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -261,14 +261,14 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
                     }}
                     onKeyPress={textInputKeyPress}
                 />
+                {!_.isEmpty(formError) && (
+                    <FormHelpMessage
+                        style={[styles.pAbsolute, styles.b0, styles.mb0, styles.ph5, styles.w100]}
```

Comment:
Absolutely positioning the error message caused https://github.com/Expensify/App/issues/32661 where the error message starts to overlap the amount input when zooming in.